### DISCUSSION TITLE
OnReading continuously fires on iOS, but not Android

### DISCUSSION THREAD

**Discussion by bb281** [shinyorg/shiny#680] [Q&A] [answered]
I've implemented the IGPSDelegate.OnReading interface method and registered the Delegate.
I set the interval to 60 seconds and I set the throttle to 50 seconds. 

The OnReading gets called about every 50/60 seconds on Android as expected, but OnReading continuously gets called on iOS. Does anyone know what might be causing this behavior/issue?

**Comment by aritchie** [comment, marked as answer]
iOS doesn't respect throttling when the app is in the foreground
